automation/taskcluster/lib/tasks.py

```diff
@@ -575,7 +575,7 @@ def _craft_raptor_task(
                 ]],
                 "env": {
                     "EXTRA_MOZHARNESS_CONFIG": json.dumps({
-                        "test_packages_url": "{}/{}/artifacts/public/build/target.test_packages.json".format(_DEFAULT_TASK_URL, mozharness_task_id),
+                        "test_packages_url": "{}/{}/artifacts/public/build/en-US/target.test_packages.json".format(_DEFAULT_TASK_URL, mozharness_task_id),
```

Comment:
I was skeptical about using these artifacts, but it turns out Raptor is happy. Then, I don't think we need to rely on the B tasks anymore \o/

Reply:
Wow, and using the new index! This is awesome!